as2/server/mail.md

```diff
@@ -0,0 +1,89 @@
+# Mail
+##### Messages that pertain to users' mail system.
+---
+### Delete Mail From Player
+Sent to clients when a postcard has been deleted from their penguin's mail.
+
+`mdp`
+
+|Argument|Type|Variadic?|
+|---|---|---|
+|newMailCount|Unsigned integer|No|
+
+##### Example
+`%xt%mdp%0%15%`
+
+### Send Mail
+Sent to clients after a send postcard request with either a successful result or an error.
+
+`ms`
+
+|Argument|Type|Variadic?|
+|---|---|---|
+|newCoinsAmount|Unsigned integer|No|
+|postCardStatus|Unsigned integer|No|
+
+##### `postCardStatus` Possibilities Table
+|Status ID|Definition|
+|---|---|
+|0|Max Mail Count Reached|
+|1|Mail Sent(Success)|
+|2|Not Enough Coins|
+
+##### Example
+`%xt%ms%0%550%1%`
+
+### Receive Mail
+Sent to client that received a postcard.
+
+`mr`
+
+|Argument|Type|Variadic?|
+|---|---|---|
+|senderUsername|String|No|
+|senderID|Unsigned integer|No|
+|mailType|Unsigned integer|No|
```

Comment:
mailType doesn't describe this argument very well - maybe artID or postcardID? This argument is responsible for the postcard art shown to the user.